gravity-forms/gw-multiple-entries-list-field.php

```diff
@@ -110,9 +120,46 @@ public function create_multiple_entries( $entry ) {
 			}
 		}
 
+		gform_add_meta( $entry['id'], 'gwmelf_parent_entry', true );
+		gform_add_meta( $entry['id'], 'gwmelf_group_entry_id', $entry['id'] );
+
+		// Update the passed entry for other filters
+		// that it may be passed to afterwards.
+		$entry['gwmelf_parent_entry']   = true;
+		$entry['gwmelf_group_entry_id'] = $entry['id'];
+
+		if ( $this->_args['append_list_data'] && $this->_args['process_feeds'] ) {
+			/**
+			 * 1st row of list data is added to the DB as a separate entry
+			 * so feed shouldn't be processed for the parent, else we get
+			 * e.g. (for GPGS) an extra row with empty list column values.
```

Comment:
Thanks for the inline comment!

Per the WordPress Coding Standards, we prefer line breaks after around 80 characters for the sake of consistency: https://developer.wordpress.org/coding-standards/inline-documentation-standards/php/#:~:text=or%20%E2%80%9Clink%20tag%E2%80%9D.-,Line%20wrapping,-DocBlock%20text%20should

Reply:
Alright, noted. Thanks!